spreadsheets.qmd

```diff
@@ -170,7 +170,7 @@ However there are a few things we might want to address in this dataset:
     ```
 
 It took us multiple steps and trial-and-error to load the data in exactly the format we want, and this is not unexpected.
-Data science is an iterative process, and the process of iteration can be even more tedious when reading data in from spreadsheets compared to other plain text, rectangular data files because humans tend to input data into spreadsheets and use them not just for data storage but also for sharing and communication.
+Data science is an iterative process, and the process of iteration can be even more tedious when reading data in from spreadsheets compared to other plain text rectangular data files because humans tend to input data into spreadsheets and use them not just for data storage but also for sharing and communication.
```

Comment:
```suggestion
Data science is an iterative process, and the process of iteration can be even more tedious when reading data in from spreadsheets compared to other plain text, rectangular data files because humans tend to input data into spreadsheets and use them not just for data storage but also for sharing and communication.
```